api/serializers.py

```diff
@@ -152,6 +153,54 @@ class QuizTitleUpdateSerializer(serializers.Serializer):
     title = serializers.CharField(required=True, allow_blank=False)
 
 
+class AddQuizSessionLogSerializer(serializers.Serializer):
+    quiz_session_code = serializers.CharField(required=True, allow_blank=False)
+    quiz_session_student_id = serializers.IntegerField(required=True, min_value=0)
+    question_multiple_choice_id = serializers.UUIDField(required=False)
+    action = serializers.CharField(required=True)
```

Comment:
You should add the other attributes/fields and make them `read_only` since we'd like to return them in the response object.